### DISCUSSION TITLE
All-in-one visualization

### DISCUSSION THREAD

**Discussion by wavoigt** [GyroGearl00se/ESPHome-Froeling-Lambdatronic_3200#15] [Ideas]
I made a complete visualization as an overview, maybe you find it useful:
(Some Entities are not provided from Froeling, these are tbd)
![froeling_visualisierung](https://github.com/GyroGearl00se/ESPHome-Froeling-Lambdatronic_3200/assets/24320737/33255425-c0ca-4ff3-8bd8-9bd329d726ca)
[froeling_visualisierung.svg.txt](https://github.com/GyroGearl00se/ESPHome-Froeling-Lambdatronic_3200/files/13981024/froeling_visualisierung.svg.txt)
[visualisierung.yaml.txt](https://github.com/GyroGearl00se/ESPHome-Froeling-Lambdatronic_3200/files/13980974/visualisierung.yaml.txt)

**Comment by Matze5593** [comment]
it's great. However, solar thermal energy is missing

**Comment by wavoigt** [comment]
It's on my todo list ;-)

**Reply by Matze5593** [reply]
I tried it once, but I've never done it. However, the Modbus values ​​would still have to be read out by the ESP
![solarthermie](https://github.com/GyroGearl00se/ESPHome-Froeling-Lambdatronic_3200/assets/55891473/c225211e-e8d1-4d5b-bfd4-8d05e350c418)

**Reply by wavoigt** [reply]
“There is always a first time” ;-)

**Comment by wavoigt** [comment]
You can add any entities to the svg and to the dashboard yaml file, for example from the solar controller.

**Comment by GyroGearl00se** [comment]
Added the AllInOne View - https://github.com/GyroGearl00se/ESPHome-Froeling-Lambdatronic_3200/commit/9b486675f16060ad457276da91fffc4d1a643db0

Thx @wavoigt - Looks pretty good :)


I don't have solar thermal on my roof. But if you manage to read out the required data and can confirm that it works as expected - I'm happy to add it into the repo. @Matze5593 

If you can tell me which values from the "B1200522_ModBus Lambdatronic 3200_50-04_05-19_de.pdf" would be required, I could try to create the corresponding entries for ESPHome and you can try it out.

**Comment by wavoigt** [comment]
I do it for my son Jan, he has a SMA controller.
I'll post my progress here.

**Comment by wavoigt** [comment]
With a slightly modification in home.css you can change text size and text color in froeling_visualisierung.svg:
(but take care if you edit the svg with incscape, it may overwrite the styles for new or modified text elements 
if this happens, you have to remove these styles manually in the text elements by using a text editor)

<pre>/* Font color */
.st9 {
  fill:#323232;
}
/* Font size */
.st11  {
  font-size:12px !important;
}
</pre>

[home .css.txt](https://github.com/GyroGearl00se/ESPHome-Froeling-Lambdatronic_3200/files/13987731/home.css.txt)

**Reply by GyroGearl00se** [reply]
https://github.com/GyroGearl00se/ESPHome-Froeling-Lambdatronic_3200/commit/e8f42706b4d378f03c60faaa791921326d7201e9

Thank you

**Comment by wavoigt** [comment]
Now i have implemented the SMA controller and Smart meter.
You have to modify the entity id's in the svg file and the ha_dashboard for your controller and meter.
![froeling_aio_sma](https://github.com/GyroGearl00se/ESPHome-Froeling-Lambdatronic_3200/assets/24320737/cb9efeb1-2697-475f-8200-35b3efab5044)
[froeling_aio_sma.svg.txt](https://github.com/GyroGearl00se/ESPHome-Froeling-Lambdatronic_3200/files/14052453/froeling_aio_sma.svg.txt)
[ha_dashboard_aio.yaml.txt](https://github.com/GyroGearl00se/ESPHome-Froeling-Lambdatronic_3200/files/14052456/ha_dashboard_aio.yaml.txt)

**Comment by Blackcherry87** [comment]
wie kann ich die Pelleteinheit entfernen?

**Reply by wavoigt** [reply]
Einfach mit Inkscape die entsprechenden Dinge löschen. 
Wenns sauber sein soll, die dazugehörigen Entities in der dashboard .yaml ebenfalls löschen.

**Comment by BlackStar153** [comment]
Hey, the image size isn't right for me. Do you have any ideas on how to make it smaller so I can see the whole thing?
<img width="690" height="411" alt="grafik" src="https://github.com/user-attachments/assets/bf41d134-7f90-4632-b6ff-d374315f6d47" />

**Reply by GyroGearl00se** [reply]
In Home Assistant you have different options when you create a Dashboard. Panel, Tile, ... (In german "Abschnitt", "Kacheln", "Panel" oder "Seitenleiste").
Depending on the Card, on your device (Computer, Tablet or Phone) you might have to pick whatever suits you the best.

I just tried a little bit out and here a suggestion for you:

<img width="518" height="509" alt="image" src="https://github.com/user-attachments/assets/9cc57dcc-aee7-4d1f-9a2d-0abbb7ae5275" />

<img width="1086" height="616" alt="image" src="https://github.com/user-attachments/assets/c3ffdde2-0f37-4a12-9ee4-9f2eba7fc38a" />

<img width="1052" height="482" alt="image" src="https://github.com/user-attachments/assets/c26d4e28-40eb-4bf0-aed3-82dce7469494" />

This results into:

<img width="1653" height="926" alt="image" src="https://github.com/user-attachments/assets/a3e607f1-1dce-49e9-a616-5dc1e45f58de" />

I hope this helps ?

**Reply by BlackStar153** [reply]
Nein, ich kann garnicht das Layout einstellen.
<img width="1476" height="1125" alt="grafik" src="https://github.com/user-attachments/assets/f8ded9dc-6d9f-4410-b209-450b487045b9" />

**Comment by wavoigt** [comment]
I'm not at my PC right now, but can't you change the overall size using Inkscape—for example, by selecting everything and resizing it?

Bin jetzt nicht am PC, aber kannst du nicht mit Incscape die Gesamtgrösse ändern, z B. alles markieren, Grösse ändern ?

**Comment by GyroGearl00se** [comment]
Ich bin mir nicht sicher ob ihr die Karte von dieser Repository verwendet (das, was HA Floorplan vorraussetzt).
Oder ob ihr inzwischen schon das neuere https://github.com/GyroGearl00se/lovelace-froeling-card in Verwendung habt.

Vielleicht macht es das ganze mit der neuen Karten einfacher ?

**Reply by BlackStar153** [reply]
Die kenne ich noch garnicht. Vlt. solte ich diese mal Probieren.
